e2e/internal/command/command.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Might be nice to have an AwsCLI struct, like we have for the CopilotCLI

Reply:
:+1:

Reply:
Maybe it's not a good idea since the reason why we have a struct for copilot is because of mocking.